### PR TITLE
Dip comments

### PR DESCRIPTION
**Pull request by realmankwon**
Loading comments : create a separate comments loading button and load it separately from the post content loading

### CODE REVIEW

**Comment by melodyzxd** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

File: config/default.json
Comment (ety001) [inline config/default.json:1]: Please don't commit your changes in default config file.

Comment (realmankwon) [reply config/default.json:1]: So sorry, I didn't know that.

Comment (ety001) [reply config/default.json:1]: you could add environment settings by the `custom-environment-variables.json` file.

**Comment by ety001** [comment]
@realmankwon Hi, please resolve conflicts, thank you.

**Review by ety001 — APPROVED** [review]

**Pull request merged**